app/api/routes/image_queries.py

```diff
@@ -171,6 +170,7 @@ async def post_image_query(  # noqa: PLR0913, PLR0915, PLR0912
                         patience_time=patience_time,
                         confidence_threshold=confidence_threshold,
                         human_review=human_review,
+                        image_query_id=image_query.id,
```

Comment:
I think this is the only change needed for associating edge predictions with escalated queries.